packit_service/worker/reporting.py

```diff
@@ -65,6 +66,10 @@ def report(
     def set_status(
         self, state: CommitStatus, description: str, check_name: str, url: str = "",
     ):
+        # required because pagure api doesnt accept, empty url
+        if isinstance(self.project, PagureProject):
+            url = "http://no.url"
```

Comment:
hm? we should find a better default: e.g. "https://wiki.centos.org/Manuals/ReleaseNotes/CentOSStream"

Reply:
Or just empty one -- we already use empty string for GitHub.

Reply:
it will be ideal, but pagure api doesnt support empty string and is checking if is valid string

Reply:
Franta, read the comment at line 69 :P

Reply:
fixed

Reply:
OK, I understand empty as `None`. Sorry